cloudinit/distros/package_management/snap.py

```diff
@@ -35,4 +35,13 @@ def install_packages(self, pkglist: Iterable) -> UninstalledPackages:
 
     @staticmethod
     def upgrade_packages():
-        subp.subp(["snap", "refresh"])
+        (out, _err) = subp.subp(
```

Comment:
parenthesis not necessary, nor is the unused variable since subp returns a named tuple